### DISCUSSION TITLE
Get webpack entry point name and reuse it in loader

### DISCUSSION THREAD

**Discussion by gottayan** [webpack/webpack#14952] [General]
https://stackoverflow.com/questions/59944550/get-webpack-entry-point-name-and-reuse-it-in-loader

**Comment by alexander-akait** [comment]
You should not do it

**Reply by alexander-akait** [reply]
there is not different between module and entry point module for loader context

**Reply by alexander-akait** [reply]
Anyway you can get the name of file using `this.resourcePath` in loader context
